Microsoft.Azure.CosmosRepository/src/Builders/PatchOperationBuilder.cs

```diff
@@ -43,8 +43,7 @@ private string GetPropertyToReplace(PropertyInfo propertyInfo)
 
             foreach (object attribute in attributes)
             {
-                JsonPropertyAttribute jsonAttribute = (JsonPropertyAttribute) attribute;
-                if (jsonAttribute is not null)
+                if (attribute is JsonPropertyAttribute jsonAttribute)
```

Comment:
Was this throwing an exception here I imagine? Since it was using a direct case rather than `as`?

Reply:
Indeed 😄